src/main/properties/datasets.properties

```diff
@@ -374,12 +376,21 @@ org.aksw.gerbil.datasets.definition.OKE_2018_TASK3.class=org.aksw.gerbil.dataset
 org.aksw.gerbil.datasets.definition.OKE_2018_TASK3.cacheable=true
 org.aksw.gerbil.datasets.definition.OKE_2018_TASK3.experimentType=RE
 org.aksw.gerbil.datasets.definition.OKE_2018_TASK3.constructorArgs=${org.aksw.gerbil.datasets.OKE_2018_TASK3.file},${org.aksw.gerbil.datasets.definition.OKE_2018_TASK3.name}
-org.aksw.gerbil.datasets.definition.OKE_2018_TASK4.name=OKE 2018 Task 3 dataset
+org.aksw.gerbil.datasets.definition.OKE_2018_TASK4.name=OKE 2018 Task 4 dataset
 org.aksw.gerbil.datasets.definition.OKE_2018_TASK4.class=org.aksw.gerbil.dataset.impl.nif.FileBasedNIFDataset
 org.aksw.gerbil.datasets.definition.OKE_2018_TASK4.cacheable=true
 org.aksw.gerbil.datasets.definition.OKE_2018_TASK4.experimentType=OKE2018Task4
 org.aksw.gerbil.datasets.definition.OKE_2018_TASK4.constructorArgs=${org.aksw.gerbil.datasets.OKE_2018_TASK4.file},${org.aksw.gerbil.datasets.definition.OKE_2018_TASK4.name}
-
+org.aksw.gerbil.datasets.definition.OKE_2018_TASK5.name=OKE 2018 Task 5 dataset
```

Comment:
I am slightly unhappy with the task 5 and 6 dataset definition since the OKE challenge 2018 simply did not had these tasks.

It would be much better, to define the `OKE 2018 Task 4 dataset` as `OKE 2018 Task 4 train` and with `KE` as experiment type. This would make the current task 5 dataset unnecessary.

The task 6 dataset should be renamed to `OKE 2018 Task 4 evaluation`.